docs/dagster-university/pages/dagster-essentials/lesson-2/requirements-and-installation.md

```diff
@@ -9,11 +9,12 @@ lesson: '2'
 To install Dagster, you’ll need:
 
 - **To install Python**. Dagster supports Python 3.9 through 3.12.
-- **A package manager like pip or poetry**. If you need to install a package manager, refer to the following installation guides:
+- **A package manager like pip, poetry or uv**. If you need to install a package manager, refer to the following installation guides:
```

Comment:
Adding in `uv` as an option since we use that other places in the repo and was what I used to complete the class

Reply:
I would capitalize "poetry" here for consistency and add a comma after it.